src/NuGet.Clients/NuGet.VisualStudio.Contracts/NuGet.VisualStudio.Contracts.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardVersion)</TargetFramework>
+    <SkipShared>true</SkipShared>
+    <PackProject>true</PackProject>
+    <Shipping>true</Shipping>
+    <IncludeInVsix>true</IncludeInVsix>
+    <IncludeNuGetSharedFiles>true</IncludeNuGetSharedFiles>
+    <RootNamespace>NuGet.VisualStudio.Contracts</RootNamespace>
+  </PropertyGroup>
```

Comment:
@nkolev92 should we set assembly version to `1.0.0.0`, like we do for `NuGet.VisualStudio.Interop`?  Although we're telling VS to create a binding redirect for this assembly, so it shouldn't matter.

Reply:
I see a static assembly version and binding redirects as 2 options to solve the same problem. 

Given that we get to choose how we implement it, I prefer the binding redirect direction. 

Do you agree?

Reply:
yes, I agree. Just making sure I wasn't missing something.